### PR TITLE
CORE-453 musl libc

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by planck-aw — APPROVED** [review]
It compiles and runs a ReplyBot 👍

**Review by planck-aw — APPROVED** [review]
It compiles and runs a ReplyBot 👍

File: Makefile [hunk 1]
```diff
@@ -56,11 +56,7 @@ PKGCONFIG_DIR=$(PREFIX)/share/pkgconfig/
 
 CARGO_FLAGS+=$(VARIANT_FLAGS)
 
-ifneq ($(filter Darwin %BSD,$(shell uname -s)),)
-    INSTALL?=install
-else
-    INSTALL?=ginstall
-endif
+INSTALL?=install
```
Comment (ip-plank-security) [inline Makefile:59]: shouldn't that statement not come *before* the infneq?

Comment (ghost) [reply Makefile:59]: > shouldn't that statement not come _before_ the infneq?

There is no `ifneq` anymore, `install` is used per default, unless overriden by the caller.

I personally would prefer to keep the upstream logic (now that I understand a little more about it), that is, using `install` on linux, `ginstall` on macOS. If that `ginstall` is problematic for homebrew users, then my 2nd preference is just use `install`, also on macOS.

Comment (ip-plank-security) [reply Makefile:59]: I see

**Review by ip-plank-security — APPROVED** [review]

**Review by ip-plank-security — APPROVED** [review]

**Review by dbeltra — APPROVED** [review]

**Pull request merged**